### PR TITLE
if landing.html exists, overwrite the index.html generated by docusaurus

### PR DESCRIPTION
**Pull request by dovholuknf**
in order to allow for easier deployment of a landing page from otherly-generated content, allow the index.html file to be stomped by process. if landing.html exists, it'll be deployed as the index.html

### CODE REVIEW

File: docusaurus/docusaurus.config.js [hunk 1]
```diff
@@ -286,6 +286,8 @@ const config = {
           alt: 'The OpenZiti logo, an open source zero trust network overlay',
           src: 'img/ziti-logo-dark.svg',
           srcDark: 'img/ziti-logo-light.svg',
+          href: 'https://openziti.io/',
```
Comment (qrkourier) [inline docusaurus/docusaurus.config.js]: Allows the button to work wherever the site is hosted, e.g., locally or Vercel, too.

```suggestion
          href: '/',
```

File: gendoc.sh [hunk 2]
```diff
@@ -181,3 +181,14 @@ echo "running 'yarn build' in ${PWD}"
...
+if test -e "${script_root}/docusaurus/build/landing.html"; then
+  echo "landing.html detected. overwriting index.html with landing.html"
+  cp "${script_root}/docusaurus/build/index.html" "${script_root}/docusaurus/build/index.original.html"
+  cp "${script_root}/docusaurus/build/landing.html" "${script_root}/docusaurus/build/index.html"
+  echo "landing.html overwritten to index.html. index.html is now index.original.html"
+fi
+
+echo " "
+echo "------------------------"
+echo "gendoc complete"
```
Comment (qrkourier) [inline gendoc.sh:194]:
```suggestion
echo "gendoc complete"

```

**Review by qrkourier — DISMISSED** [review]

File: docusaurus/docusaurus.config.js (hunk 1 above)
Comment (dovholuknf) [reply docusaurus/docusaurus.config.js]: please put a branch up for this and verify it works. i want to commit what we have as is to make sure this process works

Comment (qrkourier) [reply docusaurus/docusaurus.config.js]: I tested it locally, but I can send a twig PR too.

Comment (qrkourier) [reply docusaurus/docusaurus.config.js]: twig PR: https://github.com/openziti/ziti-doc/pull/629

**Review by qrkourier — CHANGES_REQUESTED** [review]
twig PR: https://github.com/openziti/ziti-doc/pull/629

File: gendoc.sh (hunk 2 above)
Comment (qrkourier) [reply gendoc.sh:194]: Missing newline at end of file @dovholuknf

**Review by qrkourier — APPROVED** [review]
Missing newline at `gendoc.sh` end of file @dovholuknf

**Review by gberl002 — APPROVED** [review]

**Pull request merged**